setup.cfg

```
@@ -38,21 +38,22 @@ packages = find:
 install_requires =
     # Please adapt to the current version of the library
     # and remove the unneeded extras:
-    ghga-service-chassis-lib[api,pubsub,mongo_connect,object_storage_dao,s3,postgresql]==0.12.0
-
+    ghga-service-chassis-lib[api,postgresql]==0.12.0
+    crypt4gh==1.6
 
 python_requires = >= 3.9
 
 [options.entry_points]
 # Please adapt to package name:
 console_scripts =
-    my-microservice = my_microservice.__main__:run
+    my-microservice = encryption_key_store.__main__:run
 
 [options.extras_require]
 dev =
     # Please adapt to the current version of the library
     ghga-service-chassis-lib[dev]==0.12.0
```

```suggestion
    ghga-service-chassis-lib[dev]==0.14.2
```

Chassis lib is currently on 0.14.2